server/src/test/scala/actors/persistent/nebo/NeboArrivalActorSpec.scala

```diff
@@ -24,6 +25,8 @@ object NeboArrivalActorTest {
 
   def props(redListPassengers: RedListPassengers, now: () => SDateLike, testProbeRef: ActorRef): Props =
     Props(new NeboArrivalActor(redListPassengers, () => SDate("2017-10-25T00:00:00Z"), Option(SDate("2017-10-25T00:00:00Z").millisSinceEpoch)) {
+      override val maybeSnapshotInterval = Option(1)
+
       override def receiveRecover: Receive = {
```

Comment:
I was thinking how best we can intercept a message without re-implementing the original function. You can do something like this which means the overridden function doesn't change but you're able to send each incoming message to the probe, and it's less code:

```
  override def receiveRecover: Receive = {
    case incoming =>
      testProbeRef ! incoming
      super.receiveRecover(incoming)
  }
```